source/sjSplitAlign.cpp

```diff
@@ -1,3 +1,6 @@
+#include "IncludeDefine.h"                                                                                                                                                     
```

Comment:
Not sure why this is showing up - I thought this had already been merged upstream? Perhaps git is just having a moment :)